src/silicone/time_projectors/extend_latest_time_quantile.py

```diff
@@ -157,6 +157,15 @@ def get_values_in_key_timepoint(idf):
                 quantiles = [0.5 if np.isnan(q) else q for q in quantiles]
             output_ts = target_df.timeseries()
             iamdf_ts = iamdf.timeseries()
+            if (
```

Comment:
Maybe refactor to something like

```
later_times_nans = (
iamdf_ts.loc[:, later_times]
                .dropna(axis=0, how="all")
                .isna()
                .sum()
                .sum()
)
if later_times_nans != 0:
```

Just so it's easier to understand what the intent is

Reply:
On that note, why do we `dropna(axis=0, how="all")` first? To drop out any timeseries which are entirely nan (and presumably get dealt with elsewhere)?

Reply:
Yes, it's quite common for some variables to never report at certain times and this is generally fine (and if not, will be caught  elsewhere). Thanks!